lib/debug.nix

```diff
@@ -56,19 +56,25 @@ rec {
   traceShowVal = x: trace (showVal x) x;
   traceShowValMarked = str: x: trace (str + showVal x) x;
   attrNamesToStr = a: lib.concatStringsSep "; " (map (x: "${x}=") (attrNames a));
-  showVal = x:
```

Comment:
It isn't clear to me that traceSeqN is an improvement over showVal for this use case. Can you provide some examples of the two's output?

Reply:
`showVal` displays the value with one layer of depth. `traceValSeqN 1` does the same, `traceValSeqN 2` does the same for a depth of two layers and so forth.

To be fair, the display is different, but I don’t think there is any information in `showVal` that cannot be seen in the output of `traceValSeqN`.